packages/react/src/components/pickers/BasePicker.test.tsx

```diff
@@ -102,6 +102,19 @@ describe('BasePicker', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('renders inline suggestions correctly', () => {
```

Comment:
I'm not sure this is is doing anything, since I don't think the suggestions callout is open by default?

Reply:
Good catch! I changed it to an explicit test to check that the callout is rendered in the picker